include/declare_enum.h

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
We should add these details to the LICENSES file

Reply:
I have included it in LICENSES